src/pages/hegel/guides/general-division-of-the-logic.mdx

```diff
@@ -0,0 +1,101 @@
+---
+title: Division of the Logic
+description: Here is a quick overview of the Science of Logic
+---
+
+## General Division of the Logic
+
+### The Major Divisions
+
+```mermaid
+mindmap
+    root((The Science Of Logic))
+        The Objective Logic
+            📙 Doctrine of Essence
+            📘 Doctrine of Being
+        The Subjective Logic
+            📔 The Doctrine of the Concept
+```
+
+_The Science of Logic_ is divided into two major parts or three doctrines. In accordance with the major parts, the _Logic_ is divided into _objective logic_ and _subjective logic_. The objective logic contains the doctrines _The Doctrine of Being_ and _The Doctrine of Essence_, and the subjective logic contains _The Doctrine of the Concept_. Each of these doctrines deal with the development of logic in predominant descriptors, such as immediate existence, mediated or reflected determinations and conceptual thought proper. In other words, each of these doctrines operate in ways that distinguish what is logically understood whether the matter is fundamentally immediate, mediated or a developmental integration of both. 
+
+Hegel warns at the very start of the section General Division of The Logic&mdash;just following the [Introduction](/hegel/guides/science-of-logic-introduction)&mdash;that these divisions between parts and doctrines do not serve to philosophically determine the content of logic ahead of time, but are rather preliminary aids to help orient the reader. They are, therefore, pedagogical in nature and the full disclosure of the actual development is only found in following each determination or concept carefully from its beginning to its end. 
+
+With that in mind, one can quickly get a basic lay of the land and get a sense of the project's key manuevers. Given that the logic will develop into `the concept`, whose logic is explicitly developmental, it becomes understood that the preceding doctrines are _moments_ of its development and are as such not as self-standing as they first were understood to be&mdash;indeed, Hegel's argument is that these categories or determinations are internally inadequate. This means that everything in the logic becomes oriented around the concept and, finally, `the idea`. 
```

Comment:
"oriented around the concept and, finally, `the idea`." - might be worth stating that also the "idea" is inadequate, in the sense that it also develops further. Otherwise, one might think that the Idea is the absolute completion of the Logic.

Reply:
Yep, done!